src/Avalonia.Diagnostics/DevTools.xaml.cs

```diff
@@ -14,12 +14,21 @@
 
 namespace Avalonia
 {
-	public static class WindowExtensions
+	public static class DevToolsExtensions
 	{
-		public static void AttachDevTools(this Control control)
+		public static void AttachDevTools(this TopLevel control)
 		{
-			Avalonia.Diagnostics.DevTools.Attach((TopLevel)control.GetVisualRoot());
+			Avalonia.Diagnostics.DevTools.Attach(control);
 		}
+
+	    public static void AttachDevToolsToTopLevelOnVisualTreeAttachment(this Control control)
```

Comment:
>Also the name

You were the one to complain about names suggesting something that method doesn't do